main/guides/getting-started/README.md

```diff
@@ -1,8 +1,11 @@
-# Your First Agoric Dapp
+# Getting Started
 In these steps you will be getting your first Agoric dapp up and running!
 
 ![Your first Agoric dapp](./assets/new_002_small2.png)
 
+## Your First Agoric Dapp - Offer Up
+The Offer Up Dapp allows users to view all listed items up for sale in a marketplace. The Dapp shows 3 items by default (maps, potions, and scrolls). As a user you are able to choose up to 3 total items (in any combination). You can then create an "offer" for an amount of at least 0.25 IST. Once you sign the transaction, you will see the 3 items in your wallet and your wallet amount deducted by the Offer amount.
```

Comment:
hm. quotes around offer... I suggest dropping them.

is that to call attention to the introduction of the term? I tend to use italics. I'd use `<dfn>` if the tools supported it better. Since we're not defining it here, we should have a forward reference to where we do defined it.

But I'd rather just leave it not marked up in this case. The ordinary dictionary meaning is good enough here.

Reply:
Quotes are gone! :)